awscli/customizations/cloudformation/deploy.py

```diff
@@ -169,6 +169,30 @@ class DeployCommand(BasicCommand):
                 ' execute it to implement your changes.'
             )
         },
+        {
+            'name': 'disable-rollback',
+            'required': False,
+            'action': 'store_true',
+            'group_name': 'disable-rollback',
+            'dest': 'disable_rollback',
+            'default': False,
+            'help_text': (
+                'Preserve the state of previously provisioned resources when '
+                'the execute-change-set operation fails.'
+            )
+        },
+        {
+            'name': 'no-disable-rollback',
+            'required': False,
+            'action': 'store_false',
+            'group_name': 'disable-rollback',
+            'dest': 'disable_rollback',
+            'default': True,
+            'help_text': (
+                'Rollback all resource changes when the execute-change-set '
```

Comment:
Please rephrase it to below, use `roll back` instead of `rollback`:
```
Roll back all resource changes when the execute-change-set operation fails.
```